index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="header-container">
```

Comment:
Please use the `header` tag as the container of the toolbar section as it's requested by the previous code reviewer. Kindly follow semantic HTML. You may read more about HTML CSS best practices [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).